### PR TITLE
feat(web): add subpath for v2 registries

### PR DESCRIPTION
**Pull request by m8vago**
Add an option to set  a subpath for v2 registries.

### CODE REVIEW

File: web/crux-ui/src/components/registries/registry-fields/v2-registry-field.tsx [hunk 1]
```diff
@@ -29,6 +29,17 @@ const V2RegistryFields = (props: EditRegistryTypeProps<EditableV2RegistryDetails
...
 
+      <DyoInput
+        className="max-w-lg"
+        grow
+        name="imageNamePrefix"
+        type="text"
+        label={t('subpath')}
+        onChange={handleChange}
+        value={values.imageNamePrefix}
+        message={errors.imageNamePrefix}
```
Comment (polaroi8d) [inline web/crux-ui/src/components/registries/registry-fields/v2-registry-field.tsx:40]: Label value (`subpath`) has to be the same as name (`imageNamePrefix`), IMHO.

File: web/crux-ui/locales/en/registries.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
   "pat": "Personal access token",
   "accessToken": "Access token",
   "url": "URL",
+  "subpath": "Subpath",
```
Comment (polaroi8d) [inline web/crux-ui/locales/en/registries.json:13]: `imageNamePrefix`

File: web/crux-ui/src/validations/registry.ts [hunk 3]
```diff
@@ -45,6 +45,7 @@ export const registrySchema = yup.object().shape({
     .labelType({
       hub: 'orgOrUser',
       google: 'organization',
+      v2: 'subpath',
```
Comment (polaroi8d) [inline web/crux-ui/src/validations/registry.ts:48]: `imageNamePrefix`

File: web/crux-ui/src/models/registry.ts [hunk 4]
```diff
@@ -308,6 +309,7 @@ export const editableRegistryToDto = (ui: EditableRegistry): CreateRegistryDto =
       const details: UpsertDetailsDto<V2RegistryDetailsBase> = {
         url: ui.url,
         public: ui.public,
+        imageNamePrefix: !ui.imageNamePrefix || ui.imageNamePrefix.trim() === '' ? null : ui.imageNamePrefix,
```
Comment (nandor-magyar) [inline web/crux-ui/src/models/registry.ts:312]: This "null or empty" comparison seems generic, might make a fine helper function (optional, just a note).

**Review by nandor-magyar — APPROVED** [review]

**Review by polaroi8d — APPROVED** [review]
Minor: mixing a little bit the `imageNamePrefix` and `subpath` in my interpretation.

**Pull request merged**